.github/workflows/pull-request.yml

```diff
@@ -28,3 +28,10 @@ jobs:
           mkdir build
           cd build
           kas build ../ci/${{ matrix.machine }}.yml
+
```

Comment:
How about using 

publish:
   needs: kas

Reply:
Sounds good. once we have a bootable image, I'll update this PR